tests/phpunit/api/v3/GeometryTest.php

```diff
@@ -336,17 +365,20 @@ public function testGetGeometryCentroid() {
    * Test Archiving a Geometry
    */
   public function testArchiveGeometry() {
+    // Create Collection for wards
```

Comment:
I think you should review these comments to make them more generic.  Certainly the tests are specific real-world examples of how the code might be used, but the test itself is more generic.  For example, this comment could be "Create collection to test the archiving function" could be used.  That it happens to be Qld Wards is neither here nor there - it could be anything.  You could even call it "Test Archive Collection".